docs/release-notes/Percona-Everest-0.10.0-(2024-05-03).md

```diff
@@ -0,0 +1,103 @@
+# What's new in Percona Everest 0.10.0
+
+To begin your journey with Percona Everest, check out the [Quickstart Guide for Percona Everest](../quickstart-guide/quick-install.md).
+
+Percona Everest is an open source cloud native database platform that helps provision and manage databases faster, scale deployments rapidly, and reduce database administration overhead. Plus, you can regain control over your data, database configuration, and DBaaS costs.
+
+Version 0.10.0 introduces the following changes:
+
+
+## Release highlights
+
+### Simplified Percona Everest upgrades
+
+!!! warning "Important"
+    You need to download CLI version >=0.10.0 for the upgrade command to work.    
+
+We're thrilled to announce that you can now upgrade your Percona Everest instance using our Command Line Interface (CLI). The CLI upgrade process is simple and straightforward, enabling you to quickly upgrade your database to the latest version.
+
+You can only upgrade one minor version at a time. For instance, you can upgrade from version 0.9.0 to version 0.10.0, but not directly from version 0.8.0 to version 0.10.0.
+
+For more information on upgrading Percona Everest, see our [documentation](https://docs.percona.com/everest/upgrade_with_cli.html).
+
+
+### Better traffic management with API rate limiting
+
+Starting with Percona Everest 0.10.0 version, we have introduced a new feature called **API rate limiting**. 
+
+API rate limiting is one of the key aspects of managing API's. With this you can set a threshold for the number of requests your API can receive within a specific period. This means you can take control and regulate the incoming traffic, mitigating the risk of server overload or abuse. 
+
+The default rate limit for Percona Everest is 100 requests per second. However, you can customize these limits according to your usage patterns and requirements. To dive deep into this feature, see our comprehensive [documentation](https://docs.percona.com/everest/use/API_rate_limit.html).
+
+
+### Added control for TLS certificate validation
+
+With the release of Percona Everest 0.10.0, you can add backup storage and monitoring instances without verifying the Transport Layer Security (TLS) certificate. TLS certificate verifies the server's certificate chain and hostname, ensuring its authenticity.
+
+
+Skipping the TLS certificate validation is necessary when using a self-signed certificate] because the certificate validation of TLS fails when using a self-signed certificate, as it hasn't been issued by a trusted authority. You can skip TLS certificate validation when there is no need to ensure the authenticity of the server holding the certificate. For instance, if you have a private network with complete control over everything within it, this identity check may not be needed.
```

Comment:
I think we should rephrase the 1st sentence. It sounds a bit odd, it's like "Skipping is necessary when using a self-signed certificate because the validation fails when using a self-signed certificate". The "when using a self-signed certificate" sounds a bit repetitive.

Reply:
@recharte I have rephrased it. Please check